frontend/types/api/notifications.ts

```diff
@@ -45,31 +45,27 @@ export interface NotificationDashboardsTableRow {
   event_types: ('validator_online' | 'validator_offline' | 'group_online' | 'group_offline' | 'attestation_missed' | 'proposal_success' | 'proposal_missed' | 'proposal_upcoming' | 'max_collateral' | 'min_collateral' | 'sync' | 'withdrawal' | 'validator_got_slashed' | 'validator_has_slashed' | 'incoming_tx' | 'outgoing_tx' | 'transfer_erc20' | 'transfer_erc721' | 'transfer_erc1155')[];
 }
 export type InternalGetUserNotificationDashboardsResponse = ApiPagingResponse<NotificationDashboardsTableRow>;
-export interface NotificationEventGroup {
-  group_name: string;
-  dashboard_id: number /* uint64 */;
-}
 export interface NotificationEventGroupBackOnline {
   group_name: string;
-  dashboard_id: number /* uint64 */;
   epoch_count: number /* uint64 */;
 }
 export interface NotificationEventValidatorBackOnline {
   index: number /* uint64 */;
   epoch_count: number /* uint64 */;
 }
 export interface NotificationValidatorDashboardDetail {
+  dashboard_name: string;
   validator_offline: number /* uint64 */[]; // validator indices
-  group_offline: NotificationEventGroup[]; // TODO not filled yet
-  proposal_missed: IndexBlocks[];
+  group_offline: string[]; // TODO not filled yet
```

Comment:
I do not get why this type is `broader` now?

Reply:
As stated above: because the dashboard name is a static part of the request and doesn't have to be part of each list item (same for group name; also see internal discussion)